monoio-http-client/src/client/key.rs

```diff
@@ -208,10 +208,10 @@ mod tests {
             .expect("unable to convert to Key");
         assert_eq!(key.port, 80);
         assert_eq!(key.host, "bytedance.com");
-        #[cfg(feature = "rustls")]
-        assert_eq!(key.server_name, Some("bytedance.com".try_into().unwrap()));
-        #[cfg(all(feature = "native-tls", not(feature = "rustls")))]
-        assert_eq!(key.server_name, Some("bytedance.com".into()));
+        // #[cfg(feature = "rustls")]
+        // assert_eq!(key.server_name, Some("bytedance.com".try_into().unwrap()));
+        // #[cfg(all(feature = "native-tls", not(feature = "rustls")))]
+        // assert_eq!(key.server_name, Some("bytedance.com".into()));
```

Comment:
Why is this commented out?

Reply:
Could you please explain it? And if you meant to delete, please not use comment.